hooks/post/30-upgrade.rb

```diff
@@ -54,6 +54,11 @@ def remove_docker_v1_content
   Kafo::Helpers.execute('foreman-rake katello:upgrades:3.0:delete_docker_v1_content')
 end
 
+def remove_gutterball
+  return true unless Kafo::Helpers.execute('rpm -q gutterball')
+  Kafo::Helpers.execute("rpm -e gutterball")
```

Comment:
Although this happens in post- and thus all the packages should have already been upgraded to tfm-